azure-kusto-data/tests/test_telemetry.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from azure.kusto.data._telemetry import KustoTracing, KustoTracingAttributes
+from azure.kusto.data._telemetry import Span, SpanAttributes
```

Comment:
you don't need to put the tests in the class

Reply:
removed class